src/v2/components/ChannelPreview/ChannelPreviewMessenger.tsx

```diff
@@ -301,7 +301,7 @@ const ChannelPreviewMessengerWithContext = <
             >
               {!!unread && (
                 <Text numberOfLines={1} style={[styles.unreadText, unreadText]}>
-                  {unread}
+                  {unread <= 99 ? unread : '99+'}
```

Comment:
probably should hard force everyone to this just because the design team wants it this way

Reply:
Makes sense!!